test/models/erratum_test.rb

```diff
@@ -126,5 +127,12 @@ def test_installable_for_systems
       refute_includes errata, @bugfix
       refute_includes errata, @enhancement
     end
+
+    def test_installable_for_systems_dev_environment
+      errata = Erratum.installable_for_systems([@errata_server_dev, @simple_server])
+      assert_includes errata, @security
+      refute_includes errata, @bugfix
+      refute_includes errata, @enhancement
+    end
```

Comment:
would add one more test:

```
    def test_installable_for_systems_dev_environment_with_repos
      #Tests issue #10681
      errata = Erratum.installable_for_systems([@errata_server_dev, @simple_server]).in_repositories(@repo)
      assert_includes errata, @security
      refute_includes errata, @bugfix
      refute_includes errata, @enhancement
    end
```

This should fail without your fix, and pass with your fix